cmd/siot/main.go

```diff
@@ -14,15 +18,56 @@ import (
 	"github.com/simpleiot/simpleiot/sim"
 )
 
+func send(portal, sample string) error {
+	frags := strings.Split(sample, ":")
+	if len(frags) != 4 {
+		return errors.New("format for sample is: 'devId:sensId:value:type'")
+	}
+
+	deviceID := frags[0]
+	sampleID := frags[1]
+	value, err := strconv.ParseFloat(frags[2], 64)
+	if err != nil {
+		return err
+	}
+
+	sampleType := frags[3]
+
+	sendSamples := api.NewSendSamples(portal, deviceID, time.Second*10, false)
+
+	err = sendSamples([]data.Sample{
+		data.Sample{
+			ID:    sampleID,
+			Type:  sampleType,
+			Value: value,
+		},
+	})
+
+	return err
+}
+
 func main() {
+	flagDebugHTTP := flag.Bool("debugHttp", false, "Dump http requests")
 	flagSim := flag.Bool("sim", false, "Start device simulator")
-	flagSimPortal := flag.String("simPortal", "http://localhost:8080", "Portal URL")
+	flagDisableAuth := flag.Bool("disableAuth", false, "Disable auth (used for development)")
```

Comment:
It might be clearer to call the flag `auth` or `enableAuth`, with a default of true.

Reply:
This personal preference, but would rather leave as is for now.